OmsAgent/Fairfax/HandlerManifest.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "name":  "OmsAgentForLinux",
+    "version": "1.3.18.5",
+    "handlerManifest": {
+      "installCommand": "omsagent_ff.py -install",
```

Comment:
@varunkumta might be able to provide more details on this, but one of the command templates is different, so there the two source files are not, in fact, identical.